lib/schemes/refresh.js

```diff
@@ -69,21 +80,80 @@ export default class RefreshScheme extends LocalScheme {
       this.options.endpoints.login
     )
 
-    // Update tokens
+    /*
+    * Assign tokens based on the `property` key provided in the `auth.strategy_name.token` or `auth.strategy_name.refreshToken` config
+    * auth:{
+    *   strategy_name:{
+    *     token:{
+    *       property:'token'
+    *     },
+    *
+    *     refreshToken:{
+    *       property:'refresh'
+    *     }
+    *   }
+    * }
+    * */
+
     this.$auth.token.set(getResponseProp(response, this.options.token.property))
     this.$auth.refreshToken.set(getResponseProp(response, this.options.refreshToken.property))
 
-    // Update client id
+    /*
+    * Assign a client id from the JSON response
+    *
+    * Scheme configuration example:
+    * auth:{
+    *   strategy_name:{
+    *     clientId:{
+    *       property:'clientid'
+    *     }
+    *   }
+    * }
+    * */
```

Comment:
I found an issue related to client id and grant type in `local` and `refresh` schemes. They were fixed in #602 .
So I think we can remove this comment. Then after merge the PR we can add the properly comment based on the changes. :)